netbox/utilities/forms/filterset_mappings.py

```diff
@@ -0,0 +1,11 @@
+# Mapping of filter form classes to their corresponding FilterSet classes
+# This enables the FilterModifierMixin to verify which lookups are actually supported
+# by checking the FilterSet's auto-generated lookup filters.
+#
+# Usage:
+#   from utilities.forms.filterset_mappings import FILTERSET_MAPPINGS
+#   from .forms.filtersets import XFilterForm
+#   from .filtersets import XFilterSet
+#   FILTERSET_MAPPINGS[XFilterForm] = XFilterSet
+
+FILTERSET_MAPPINGS = {}
```

Comment:
We should avoid introducing one-off globals to be used for making associations in this manner. Instead, we can leverage the [application registry](https://netboxlabs.com/docs/netbox/development/application-registry/) to correlate FilterSets to their corresponding models; the models are already set on the filter forms. Then you could do something like this:

```python
@register_filterset
class CircuitFilterSet(PrimaryModelFilterSet, TenancyFilterSet, ContactModelFilterSet):
```

```python
from netbox.registry import registry
filterset = registry['filtersets'].get(f'{app_label}.{model}')
```

We take a similar approach to map models to their search indexes (`registry['search']`).